src/setup/stateful.rs

```diff
@@ -0,0 +1,72 @@
+//! Contains code to start and stop a node with preloaded ledger data.
+
+use std::{
+    net::{Ipv4Addr, SocketAddr, SocketAddrV4},
+    path::PathBuf,
+    process::Child,
+};
+
+use crate::setup::{config::NodeMetaData, process, process::wait_for_start};
+
+const STATEFUL_PORT: u16 = 5005;
```

Comment:
why this port (it should be documented if it's meaningful)? shouldn't it be in the consts module?

Reply:
Moved to consts and commented.